src/components/SizeDropDown/SizeDropDown.js

```diff
@@ -0,0 +1,31 @@
+// import { useForm } from "react-hook-form";
+import { useLocation } from "react-router-dom";
+import "./SizeDropDown.css";
+
+export default function SizeDropDown({ value, onChange, className }) {
+  const location = useLocation();
+  // const { register } = useForm();
+
+  return (
+    <label htmlFor="size">
+      <select
+        id="size"
+        name="company_size"
+        type="text"
+        value={value}
+        onChange={onChange}
+        className={className}
+        placeholder="Select company size"
+        // {...register("company_size")}
+      >
+        {className === "dropDown" || location === "/add-agency" || (
```

Comment:
This seems needlessly complex 

- For the search displaying "Select company size" is fine (right?)
- For add  displaying "Select company size" is also fine (right?) of course if you don't select something you should get a validation error
- For edit there should already be a size to display, you can leave the option: "Select company size" in, but the validation should give an error if you select that one.

If you don't want to do it this way you can add props.options and pass an array of <option> elements where you render this form (but this is way more complex I think)

The reason I am not a fan of: ` {className === "dropDown" || location === "/add-agency" || (`

1. It makes my brain hurt .. (I am a fragile person)
2. If I am going to make the next feature, do I need to add another exception here if I want to reuse this?
3. I have to complain about _something_ right?